### PR TITLE
Updating DNS Shuttering documentation to match new method

### PR DESCRIPTION
**Pull request by MartyFox**
### Jira link (if applicable)
https://tools.hmcts.net/jira/browse/DTSPO-16428

### Change description ###
Updating DNS Shuttering documentation to match new method

### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

File: source/cloud-native-platform/path-to-live/shutter.html.md.erb [hunk 1]
```diff
@@ -1,54 +1,126 @@
...
\ No newline at end of file
+See [azure-shutter-pages](https://github.com/hmcts/azure-shutter-pages) repository for examples, where there is shutter pages for further directories beyond the main URL.  For example: [et-pet](https://github.com/hmcts/azure-shutter-pages/tree/master/et-pet)
+
+
+
+
+
+Use the following steps to enable shuttering:
+
+Clone the azure-public-dns repository
```
Comment (thomast1906) [inline source/cloud-native-platform/path-to-live/shutter.html.md.erb:108]: do we want to number the steps?

Comment (MartyFox) [reply source/cloud-native-platform/path-to-live/shutter.html.md.erb]: Good spot, those shouldnt have been in that file, they were copied from my notes.
Remove and numbering added to main steps.

File: source/cloud-native-platform/path-to-live/shutter.html.md.erb [hunk 2]
```diff
@@ -1,54 +1,105 @@
...
+
+7. If the shutter config exists, changing the shutter value from false to true is all that is required to enable shuttering for this service.
+
+- <details>
+    <summary>What happens when you change the shutter value</summary>
+    <p>Setting <code>shutter: true</code> will enable the shutter page and redirect all the traffic to Azure CDN.<br>
+    Setting <code>shutter: false</code> will disable the shutter page and redirect all the traffic back to Azure Frontdoor.</p>
+    </details>
+
+8. Create a PR with the changes and have it reviewed. Once merged this will update the exist record to point to the Azure Static Web App endpoint.
```
Comment (reespozzi) [inline source/cloud-native-platform/path-to-live/shutter.html.md.erb:63]: Pedantic sorry, but this doesn't seem to render correctly :
image

File: source/cloud-native-platform/path-to-live/shutter.html.md.erb [hunk 3]
```diff
@@ -1,54 +1,105 @@
...
+A record types have some specific requirements with regard to shuttering. A records typically only allow IP addresses to be used BUT within Azure it is possible to direct A records to Azure resources, known as alias records.
+This means we can direct A record types to Azure services such as Azure Front Door or Azure Web Apps.
+
+The way DNS has been setup for this project, an A record can have one of the following set:
+
+- **record** - either a single IP or a list of IP addresses
+- **shutter_resource_id** - an Alias to an Azure resource specifically used for shuttering
+- **alias_target_resource_id** - an Alias to an Azure resource if not IP is used, can also be used for shutting but preference should be shutter_resource_id
+
+For shuttering purposes if you need to shutter an A record it must> have one of the following settings in the A record configuration:
```
Comment (reespozzi) [inline source/cloud-native-platform/path-to-live/shutter.html.md.erb:88]:
```suggestion
For shuttering purposes if you need to shutter an A record it must have one of the following settings in the A record configuration:
```

File: source/cloud-native-platform/path-to-live/shutter.html.md.erb [hunk 4]
```diff
@@ -1,54 +1,105 @@
...
+
+There are two record types that allow shuttering to be enabled.
+These are:
+
+- A records
+  - The "A" stands for "address" and this is the most fundamental type of DNS record. It indicates the IPv4 address of a given domain.
+- CNAME records
+  - A "canonical name" (CNAME) record points from an alias domain to a "canonical" domain. A CNAME record is used in lieu of an A record, when a domain or subdomain is an alias of another domain. All CNAME records must point to a domain, never to an IP address.
+
+### Shuttering A record types
```
Comment (reespozzi) [inline source/cloud-native-platform/path-to-live/shutter.html.md.erb:77]: Should there also be something small in this section about _how_ to shutter? Maybe just pointing to above section and some rewording?

CNAME bit has this for instance, but A records seems more like info & pre-reqs
 ```
CNAME records are much easier to shutter, for each record an equivalent shutter config must exist, as detailed in the Enable shuttering with a DNS swap section.

Simply enable or disable the shutter using True/False values
```

File: source/cloud-native-platform/path-to-live/shutter.html.md.erb (hunk 2 above)
Comment (MartyFox) [reply source/cloud-native-platform/path-to-live/shutter.html.md.erb]: Should be fixed.

**Review by reespozzi — APPROVED** [review]

**Pull request merged**